spec/importer/mahonia_mapper_spec.rb

```diff
@@ -44,6 +44,36 @@
     end
   end
 
+  it "provides an embargo date field as a Mahonia::EdtfLiteral" do
+    mapper.metadata = { "embargo_date" => ["1997-02-01 00:00"] }
+
+    expect(mapper.embargo_release_date.first).to be_an_instance_of(Mahonia::EdtfLiteral)
```

Comment:
@no-reply this seems sufficient for validating that we import date fields so that they are persisted correctly, doesn't it? A new Validator doesn't seem to be what we need to ensure this.